package.json

```diff
@@ -46,7 +46,7 @@
     "@ledgerhq/hw-transport-http": "^5.51.1",
     "@ledgerhq/hw-transport-node-hid-singleton": "^5.51.1",
     "@ledgerhq/ledger-core": "6.11.3",
-    "@ledgerhq/live-common": "19.9.0",
+    "@ledgerhq/live-common": "19",
```

Comment:
19.10.0? @gre

Reply:
it's not a problem. only yarn.lock is the source of trust for versions. "19" is equivalent to `^19.0.0` which is legit match. but package.json is not really the truth.